python/taichi/lang/expr.py

```diff
@@ -114,7 +114,7 @@ def fill(self, val):
         from .meta import fill_tensor
         fill_tensor(self, val)
 
-    @deprecated('tensor.parent()', 'tensor.snode().parent()')
+    #@deprecated('tensor.parent()', 'tensor.snode().parent()')
```

Comment:
So we no longer deprecate this?

Reply:
Yes, since `tensor.parent()` returns a `tensor` while `tensor.snode().parent()` return a snode, it seems this is irreplaceable and many test is using this..